mlir/lib/Dialect/SparseTensor/IR/SparseTensorDialect.cpp

```diff
@@ -13,6 +13,7 @@
 #include "mlir/IR/Matchers.h"
 #include "mlir/IR/OpImplementation.h"
 #include "llvm/ADT/TypeSwitch.h"
+#include <iostream>
```

Comment:
Is this used? Maybe I missed it, but I'm unsure if this include is necessary.

Reply:
Good catch! That was a leftover from debugging.